backend/core/src/email/email.service.ts

```diff
@@ -224,34 +255,115 @@ export class EmailService {
     )
   }
 
-  public async listingOpportunity(user: User, appUrl: string) {
-    const jurisdiction = await this.getUserJurisdiction(user)
-    void (await this.loadTranslations(jurisdiction, user.language))
+  public async listingOpportunity(listing: Listing) {
+    const jurisdiction = await this.jurisdictionService.findOne({
+      where: {
+        id: listing.jurisdiction.id,
+      },
+    })
+    void (await this.loadTranslations(jurisdiction, Language.en))
```

Comment:
question: do we need the `void` here?

Reply:
I don't think it's needed, but it is the pattern in this file so I'm a bit worried to pull it out for this for some weird edge case. When we migrate to prisma we won't use this pattern so I'd vote to get rid of the void at that point